### ISSUE TITLE
A11y testing

### ISSUE THREAD

**Issue by shinokada** [themesberg/flowbite-svelte#62]
## Summary

We can use axe-cli to test for A11y. But it will be nice to automate it.

### Basic example

```
npm install @axe-core/cli -g
axe http://localhost:3000/dropdowns/image
```

### Motivation

The library and the website should be A11y friendly.
